output/benchmarking/latest_subset_forecasts_by_location.tsv

```diff
@@ -1,7 +1,7 @@
 crps_hosp	crps_ww	bias_hosp	bias_ww	ae_hosp	ae_ww	location	wweval_commit_hash	wwinference_version	time_stamp
-1.0465392954054191	1.0437418123363529	-0.052574561403508774	-0.05236184210526316	1.430541441794062	1.4249102564297214	AK	d23b28a	v0.1.0	2024-10-24T17:48:21Z
-11.624616908337584	11.001390757513024	0.23765679824561403	0.30562280701754385	15.2169931884645	14.921735155883097	MA	d23b28a	v0.1.0	2024-10-24T17:48:21Z
-1.9743211076933298	1.7647838214371094	0.10642543859649123	0.11702741228070176	2.6690277913463327	2.360288987441267	NH	d23b28a	v0.1.0	2024-10-24T17:48:21Z
-17.371192144075163	17.18502224109214	0.16711842105263158	0.24489583333333334	22.505788770781994	22.54254318981423	NJ	d23b28a	v0.1.0	2024-10-24T17:48:21Z
-3.5381177843863316	4.552876071939314	0.0430734649122807	0.31171600877192984	4.9099199893294525	6.397181233712242	WA	d23b28a	v0.1.0	2024-10-24T17:48:21Z
-7.110957447979565	7.109562940863587	0.10033991228070176	0.18538004385964912	9.346454236343268	9.529331764656112	all	d23b28a	v0.1.0	2024-10-24T17:48:21Z
+1.0376521512900907	1.0118669366748054	-0.036991228070175436	-0.007656798245614037	1.4245526802625763	1.4264265157766278	AK	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
+24.626800317489625	19.49484775911998	0.2060657894736842	0.4117872807017544	30.689131156635455	24.783400147179314	MA	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
+2.9839338399865656	1.8872799196790222	-0.01010087719298246	0.04486293859649123	4.248055546108953	2.2765310519617756	NH	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
+50.50136383077716	41.39684785751621	0.1817642543859649	0.2782510964912281	62.26077007486285	51.60308220739496	NJ	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
+4.468227694679483	10.723086482359754	-0.015425438596491228	0.33135855263157893	6.534705999620968	14.015772038017861	WA	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
+16.723595566844583	14.902785791069954	0.0650625	0.2117206140350877	21.03144309149816	18.821042392066108	all	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
```

Comment:
Looks to me like things are particularly worse in NJ, MA, WA on 10-16 and 2024-01-08. Will make a bunch of comparison figures in a cdcent/cfa-forecast-renewal-ww